packages/components/pagination/src/components/sizes.vue

```diff
@@ -41,6 +41,10 @@ const paginationSizesProps = buildProps({
     default: '',
   },
   disabled: Boolean,
+  size: {
+    type: String,
+    default: 'default',
+  },
```

Comment:
Can we use the prop defined in `useSizes`?

Reply:
No, the size only have default and small